app/metrics.yaml

```diff
@@ -0,0 +1,75 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+$schema: moz://mozilla.org/schemas/glean/metrics/1-0-0
+
+nav:
+  browser_menu:
+    type: string
+    description: >
+      String of the item clicked in the menu which includes: Back, Forward, Reload, Exit Firefox
+    send_in_pings:
+      - baseline
+    lifetime: user
+    bugs:
+      - N/A
+    data_reviews:
+      - N/A
+    notification_emails:
+      - katglazko@gmail.com
+    expires: 2100-01-01
+
+  browser_back_controller:
+    type: event
+    description: >
+      String of the option clicked on the hardware controller which includes: Back
+    extra_keys:
+      source_of_click:
+        description: Explains where the action came from
```

Comment:
For things like extras, exhaustive values (or ranges) is important - the goal is for this to be a non-technical output so someone can easily decide "yes this is okay to collect" or "no some of these values are too sensitive to collect".